c/busybox-1.22.0/busybox_sv_comp-uname.h

```diff
@@ -0,0 +1,33 @@
+int uname(struct utsname *uts)
```

Comment:
```suggestion
extern char __VERIFIER_nondet_char(void);
extern int __VERIFIER_nondet_int(void);

int uname(struct utsname *uts)
```

Reply:
Why add these declarations? These are always present due to the "busybox_sv_comp.h".

Reply:
I'm fine with that, but then `isatty` shouldn't be doing so either.